src/storage-adapters/mongo/index.js

```diff
@@ -85,6 +85,12 @@ export default class MongoAdapter {
     };
   }
 
+  /* eslint-disable */
```

Comment:
Why do we disable eslint here? This is smelly 😉

Reply:
Because as the function is not implemented, there are no `yield` inside and Eslint cries.